ArcadiaParties-SPA/src/app/app.component.html

```diff
@@ -1,3 +1,7 @@
-<h1>WELCOME</h1>
-<h1> Title is: {{ title$ | async }} </h1>
- 
\ No newline at end of file
+<mat-toolbar color="primary" class="header-container">
```

Comment:
Does material library set style for toolbar by itself?

Reply:
Yeah, but I've chosen this particular color style from the available.

Reply:
Yeah, but chose this particular color style from the available. No need in this, we can do it later.